bin/slt-changelog.js

```diff
@@ -0,0 +1,204 @@
+#!/usr/bin/env node
+'use strict';
+
+var _ = require('lodash');
```

Comment:
It was sorted in my imaginary ASCII alternative where `_` is before `A`! ...mostly :blush: